.github/workflows/tests.yml

```diff
@@ -0,0 +1,200 @@
+name: Tests
+
+on:
+  push:
+    branches: [ dev ]
+  pull_request:
+    branches: [ dev ]
+
+jobs:
+
+  phpunit:
+
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+         image: postgres:9.6-alpine
+         options: --name=postgres
+         env:
+           POSTGRES_DB: testing
+           POSTGRES_USER: talentcloud
+           POSTGRES_PASSWORD: talentcloud
+         ports:
+          - 5432:5432
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: shivammathur/setup-php@v2
+      with:
+        php-version: 7.2
+        extensions: dom, curl, libxml, mbstring, zip, pcntl, pdo, bcmath, soap, intl, gd, exif, iconv, imagick
+        tools: composer:v1
+
+    - name: Copy .env
+      run: cp .env.travis .env
```

Comment:
Should add a new `.env` for Github actions, or rename it, for the sake of not confusing ourselves and others in 8 months